_data/harnesses/dynamic-output-topic/kstreams.yml

```diff
@@ -0,0 +1,177 @@
+dev:
+  steps:
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/dynamic-output-topic/kstreams/markup/dev/init.adoc
+
+    - title: Get Confluent Platform
+      content:
+        - action: make_file
+          file: docker-compose.yml
+          render:
+            file: tutorials/dynamic-output-topic/kstreams/markup/dev/make-docker-compose.adoc
+
+        - action: execute_async
+          file: tutorial-steps/dev/docker-compose-up.sh
+          render:
+            file: tutorials/dynamic-output-topic/kstreams/markup/dev/start-compose.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/wait-for-containers.sh
+          render:
+            skip: true
+
+    - title: Configure the project
+      content:
+        - action: make_file
+          file: build.gradle
+          render:
+            file: tutorials/dynamic-output-topic/kstreams/markup/dev/make-build-file.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/gradle-wrapper.sh
+          render:
+            file: tutorials/dynamic-output-topic/kstreams/markup/dev/make-gradle-wrapper.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/make-configuration-dir.sh
+          render:
+            file: tutorials/dynamic-output-topic/kstreams/markup/dev/make-config-dir.adoc
+
+        - action: make_file
+          file: configuration/dev.properties
+          render:
+            file: tutorials/dynamic-output-topic/kstreams/markup/dev/make-dev-file.adoc
+            
+    - title: Create a schema for the model obect
+      content:
+        - action: execute
+          file: tutorial-steps/dev/make-avro-dir.sh
+          render:
+            file: tutorials/dynamic-output-topic/kstreams/markup/dev/make-avro-dir.adoc
+
+        - action: make_file
+          file: src/main/avro/order.avsc
+          render:
+            file: tutorials/dynamic-output-topic/kstreams/markup/dev/make-avro-schema.adoc
+
+        - action: make_file
+          file: src/main/avro/completed-order.avsc
+          render:
+            file: tutorials/dynamic-output-topic/kstreams/markup/dev/make-avro-schema.adoc
```

Comment:
@bbejeck this is redundant w/ the lines above, causing duplicates in the rendered instructions

Reply:
ack - good catch!